pages/index.js

```diff
@@ -37,21 +40,71 @@ export default function Home() {
       </div>
       <div className="container flex flex-wrap">
         <div className="mt-4 mb-4 grid lg:grid-cols-3 sm:grid-cols-2 grid-flow-row gap-3 md:gap-10">
-          {data.map(function (d, idx) {
-            return (
-              <TopicBox
-                key={idx}
-                title={d.title}
-                body={d.body}
-                image={d.image}
-                imgalt={d.imgalt}
-                url={d.url}
-                subtopics={d.subtopics}
-              />
-            );
-          })}
+          {topicsData.map(
+            ({ title, body, image, imgalt, url, subtopics }, idx) => {
+              return (
+                <TopicBox
+                  key={idx}
+                  title={title}
+                  body={body}
+                  image={image}
+                  imgalt={imgalt}
+                  url={url}
+                  subtopics={subtopics}
+                />
+              );
+            }
+          )}
         </div>
       </div>
     </Layout>
   );
 }
+
+export async function getStaticProps(language) {
+  let topicsData = [];
+  let errorCode = false;
+
+  //
+  // IF content enabled get the data from the api
+  //
+
+  if (process.env.NEXTJS_CONTENT_ENABLED) {
```

Comment:
One option would be to assume we use the API if the URL exists here, so we could put 

`if (process.env.NEXTJS_CONTENT_API)`, and then if it exists we get it from the API.